userCode/lib/lakefs.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2025 Lincoln Institute of Land Policy
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Optional
+import lakefs
+from lakefs import Client
+
+from userCode.lib.env import (
+    LAKEFS_ACCESS_KEY_ID,
+    LAKEFS_ENDPOINT_URL,
+    LAKEFS_SECRET_ACCESS_KEY,
+)
+
+
+class LakeFSClient:
+    """Helper client for abstracting lakefs operations"""
+
+    def __init__(self, repository: str):
+        self.lakefs_client = Client(
+            host=LAKEFS_ENDPOINT_URL,
+            username=LAKEFS_ACCESS_KEY_ID,
+            password=LAKEFS_SECRET_ACCESS_KEY,
+        )
+        self.repository = repository
```

Comment:
```suggestion
        self.repository = lakefs.repository(repository, client=self.lakefs_client)
```
Unless there is a reason you are instantiating this in every function?

Reply:
Good catch. Fixed